packages/frontend/src/hooks/useClaimingEndTime.ts

```diff
@@ -0,0 +1,19 @@
+import { useCall } from '@usedapp/core'
+import moment from 'moment'
+
+import { useDevconContract } from './contract'
+
+export function useClaimingEndTime() {
+  const devconContract = useDevconContract()
+
+  const { value, error } =
+    useCall(
+      devconContract && {
+        contract: devconContract,
+        method: 'claimingEndTime',
+        args: [],
+      }
+    ) ?? {}
+  const claimingEndTime = value && moment.unix(value[0].toNumber()).format('DD.MM.YYYY')
```

Comment:
Maybe return undefined when there is no response from contracts and later on handle this undefined, replacing end time with `-` similar to `LeftTime` `EndTime` in main view?